src/popup/popup.html

```
@@ -0,0 +1,298 @@
+              data-key="blockKeywords"
+            />
+            <span class="switch" aria-hidden="true"></span>
+          </label>
+          <div class="keywords-drawer" id="keywords-section">
+            <div class="drawer-meta">
+              <span class="drawer-label">LIST</span>
+              <span class="hint" id="keyword-count">0 / 50</span>
+            </div>
+            <form class="keyword-form" id="keyword-form">
+              <input
+                type="text"
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)